cmd/snapctl/commands.go

```diff
@@ -117,6 +117,21 @@ var (
 						flRunning,
 					},
 				},
+				{
+					Name: "config",
```

Comment:
I thought we can define plugin specific config inside task manifest although that's task specific.  If answer is yes, how that will have effect on the global plugin config. This topic may not related to your fix.